terraform.tfvars.example

```diff
@@ -102,4 +102,5 @@ attached_drg_id = null
 #subnets = {
 #  sub1 = {name = "subnet1",cidr_block = "10.0.4.0/24"}
 #  sub2 = {cidr_block="10.0.5.0/24",type="private"}
+#  sub3 = {cidr_block="10.0.6.0/24",availability_domain="Uocm:PHX-AD-1"}
```

Comment:
You should pass an AD number and then dynamically look this up. We cannot expect users to know this format.

Reply:
Updated the code with AD number instead of AD name

Reply:
Update the code with AD number instead of AD name